pages.hi/common/ln.md

```diff
@@ -0,0 +1,16 @@
+# ln
+
+> फाइलों और निर्देशिकाओं के लिंक बनाता है।
+> अधिक जानकारी: <https://www.gnu.org/software/coreutils/ln>।
+
+- किसी फ़ाइल या निर्देशिका के लिए एक प्रतीकात्मक लिंक बनाएँ:
+
+`ln -s {{फ़ाइल या निर्देशिका / का / पथ}} {{लिंक / का / पथ}}`
```

Comment:
I'm not sure whether to put spaces between Hindi characters and slashes. In [chsh](https://github.com/tldr-pages/tldr/blob/main/pages.hi/common/chsh.md), the author didn't use spaces, please let us know which is the correct way in Hindi. In tldr, we tend to not put spaces near slashes.

Reply:
I have removed the spaces, they could be confusing, thanks for the suggestion